fitsio/src/hdu.rs

```diff
@@ -1098,6 +1194,16 @@ mod tests {
         assert_eq!(intcol_data[49], 12);
     }
 
+   #[test]
+    fn test_cards() -> Result<()> {
+        let mut f = FitsFile::open("../testdata/full_example.fits").unwrap();
+        let primary_hdu = f.hdu(0).unwrap();
+        for card in primary_hdu.cards(&mut f).unwrap() {
+            println!("{:8} = {:10} - {}", card.name()?, card.str_value()?, card.comment()?);
```

Comment:
Of course this test doesn't actually test anything, let's revisit when we have agreement on the overall design.

Reply:
No, indeed. Just wanted proof that the code could run without hitting a lifetime snag or similar.